DESCRIPTION

```diff
@@ -1,14 +1,13 @@
 Package: meetupr
 Type: Package
 Title: Meetup API
-Version: 0.0.1
+Version: 0.0.2
```

Comment:
Can you change the version to 0.1.0?  We have already changed the version on Lucy's branch to 0.1.0.